charter.md

```diff
@@ -1,134 +1,159 @@
 
-## Immediate Action items
-
-1.  An introduction email shall be sent out by the Chair/Co-Chair to the Darn-ddwg mailing list, with this charter.
-2.  An email shall be sent out detailing the current status of data distribution, including the mirror.
-3.  Documentation shall be done on the Primary SuperDARN Data Mirror.
-4.  Discussion among all members should happen regarding the “Points for Consideration” below.
-
-## Points for Consideration
-
-1.  Are there problems with accessing files from certain institutions?
-2.  How do we become aware of, and deal with, gaps in the distribution?
-3.  Have bad files been passed to the general distribution? Should these be identified and 
-corrected, deleted, or somehow flagged? (Should be less of a problem after Oct. 1, 2013)
-4.  How do we make rawacf files available on a next-day (or better) basis?
-5.  Do we have issues with the amount of data and storage capacity?
-6.  The files currently distributed by U Sas are apparently not identical to those that were 
-provided by the PI institutions. Why is this, and is it a problem?
-7.  Should the Data Mirror set up at U Sas eventually replace distribution of data by hard drives?
-8.  How do we ensure that Virginia Tech has the most complete data archive? There are known gaps 
-within the Virginia Tech data archive across several radars (April 29 – May 6, 2008; 
-July 30, 2008; December 30, 2008)
-9.  How do we ensure that much older data that is corrected by an institution is distributed 
-throughout the SuperDARN community? For example, if a flaw is found in 2009 Blackstone data and 
-it is corrected, how do we ensure the corrected data is distributed throughout the PI institutions?
-10. Building upon \#4, how can the community quickly scrutinize/analyze data to ensure, at least 
-at first glance, proper radar operation? If a days worth of data contains 10 return-cells per beam 
-per hour, is the radar broken? Hopefully this will help keep faulty data from quickly being 
-distributed and lessen the need for my \#2.
-11. How do we process different/irregular file formats into the distribution? The University of 
-Alaska Fairbanks radars have begun producing multi-channel data that cannot utilize the channel 
-variable in the dmap header due to stereo-ops logic in the rst code that breaks. So, to keep things 
-sensible, the channel information is being stored into the filename. As well, older radars still 
-produce data in the old fit and dat file formats. Which do we use as a basis for converting to 
-the new file format?
+## Raising Issues
+
+Anyone can raise issues to the DDWG. The order of preference for raising issues is:
+ 1. Via the github repository here: <https://github.com/SuperDARN/DDWG/issues/new/choose>
+ 2. Via an email to the DDWG mailing list: <darn-ddwg@isee.nagoya-u.ac.jp>
+ 3. Via an email to the chair or co-chair(s)
+
+## Blacklisting policy
+
+TODO
+
+## File checking policy 
+
+TODO
+
+## TODO: OTHER POLICIES
```

Comment:
TODO: Updating files policy
TODO: Commissioning data policy -> Need to make sure that this is communicated to RST group, hdw.dat/radar.dat group
TODO: Removing files policy